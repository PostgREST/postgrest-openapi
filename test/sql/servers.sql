-- shows the default server
select jsonb_pretty(get_postgrest_openapi_spec('{test}')->'servers');

-- shows the custom server url specified by the user
select jsonb_pretty(get_postgrest_openapi_spec('{test}', 'https://www.example.com/api/')->'servers');