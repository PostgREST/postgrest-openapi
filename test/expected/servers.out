-- shows the default server
select jsonb_pretty(get_postgrest_openapi_spec('{test}')->'servers');
             jsonb_pretty             
--------------------------------------
 [                                   +
     {                               +
         "url": "http://0.0.0.0:3000"+
     }                               +
 ]
(1 row)

-- shows the custom server url specified by the user
select jsonb_pretty(get_postgrest_openapi_spec('{test}', 'https://www.example.com/api/')->'servers');
                 jsonb_pretty                  
-----------------------------------------------
 [                                            +
     {                                        +
         "url": "https://www.example.com/api/"+
     }                                        +
 ]
(1 row)

